src/loc/pt-pt.ts

```diff
@@ -2,6 +2,10 @@ declare var define: any;
 
 define([], () => {
   return {
+MyTeamsLoadingMessage: "loading you teams",
+MyTeamsMessageError: "Something went wrong while loading your teams, please try later or refresh browser",
+
```

Comment:
Is MyTeamsMessageDontHaveTeams missing here?

Reply:
done